File: LegoSCart/asplego/Models/UserOrder.cs
using System;
namespace Asplego.Models
{
    public class UserOrder
    {
        public string Ordertime;
        public string Useraccount;
        public int LegoId;
        public int Quantity;
        public int Payaccount;

        public UserOrder(string ordertime, string useracc, int legoid, int quantity, int payacc)
        {

            Ordertime = ordertime;
            Useraccount = useracc;
            LegoId = legoid;
            Quantity = quantity;
            Payaccount = payacc;


        }

    }
}

File: LegoSCart/LegoShop/src/app/shared/models/Userorder.ts
export class Userorder{
    Ordertime:string;
    Useraccount:string;
    LegoId: number;
    Quantity: number;
    Payaccount: number;

    constructor(ordertime:string,useracc:string,legoid:number,quantity:number,payacc:number){
        this.Ordertime=ordertime;
        this.Useraccount=useracc;
        this.LegoId=legoid;
        this.Quantity=quantity;
        this.Payaccount=payacc;
    }
}
